src/index.ts

```diff
@@ -27,37 +27,40 @@ export type Options = {
   strict?: boolean;
 };
 
-export default function destr(value: any, options: Options = {}): any {
+export default function destr<T = unknown>(
+  value: any,
```

Comment:
Not to be extra difficult, but should this be an `unknown` as well to improve `destr`'s own type safety? By treating the parameter as `any`, the type checking within this function block is effectively disabled 🤔

Reply:
PR welcome to refactor 👍🏼